ios-base/DroidKaigi 2020/About/Views/AboutViewController.swift

```diff
@@ -0,0 +1,165 @@
+import ios_combined
+import SafariServices
+import UIKit
+
+class AboutViewController: UITableViewController {
+    private enum CellType {
+        case description, access, staff, policy, license, version
+    }
+
+    private let cells: [CellType] = [.description, .access, .staff, .policy, .license, .version]
+
+    static func instantiate() -> AboutViewController {
+        guard let viewController = UIStoryboard(name: "AboutViewController", bundle: .main).instantiateInitialViewController() as? AboutViewController else { fatalError() }
+        return viewController
+    }
+
+    override func viewDidLoad() {
+        title = L10n.aboutDroidKaigi
+    }
+}
+
+extension AboutViewController {
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        cells.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch cells[indexPath.row] {
+        case .description:
+            return dequeueReusableCell(withIdentifier: AboutCell.Identifier.description, for: indexPath) { cell in
+                cell.titleLabel.text = L10n.whatIsDroidKaigi
+                let detail = L10n.droidKaigiDescription
+                let paragraphStyle = NSMutableParagraphStyle()
+                paragraphStyle.lineHeightMultiple = 1.1
+                let attributes: [NSAttributedString.Key: Any] = [.kern: 0.5, .paragraphStyle: paragraphStyle]
+                cell.detailLabel?.attributedText = NSMutableAttributedString(string: detail,
+                                                                             attributes: attributes)
+                cell.onButtonTapped = { [weak self] button in
+                    guard let self = self else { return }
+                    let urlString: String
+                    switch button {
+                    case .twitter:
+                        urlString = "https://twitter.com/DroidKaigi"
+                    case .youtube:
+                        urlString = "https://www.youtube.com/channel/UCgK6L-PKx2OZBuhrQ6mmQZw"
+                    case .medium:
+                        urlString = "https://medium.com/droidkaigi"
+                    }
+                    guard let url = URL(string: urlString) else { return }
+                    self.presentSafariViewController(with: url)
+                }
+            }
+        case .access:
+            return dequeueReusableCell(withIdentifier: AboutCell.Identifier.icon, for: indexPath) { cell in
```

Comment:
Please make this cell identifier to `access` ? 🙏

Reply:
I use AboutCell.Identifier to identify the type of layout.
Maybe I should remove the hard-cord image in the storyboard and set it here?

Reply:
Oh, sorry. In this case, I wanted to say that the identifier "icons" did not match. So I suggested here should change `icons` to `access`.

Reply:
Sorry about the inconvenience.
Did you mean you understand my thought after my explanation and this conversation is resolved?
If so, please ignore the content below.

If not I think I still don't understand what you meant.
Please let me describe my thought.
``` swift
// Use to identify the type of layout
struct Identifier {
    static let description = "description"
    static let icon = "icon" // A cell with an icon
    static let basic = "basic"
    static let detail = "detail"
}

// Use to identify the cell (There may be a better name)
private enum CellType {
    case description, access, staff, policy, license, version
}
```
So it may be better to remove the hard-cord image in the storyboard and set it here to clarify that the icon cell is just a kind of cell, not a specific cell.
And there may be a better name for `CellType`.
I cannot tell there is a need to change `icon` to `access`.
Please tell me you want me to change which `icon`?

Reply:
I've understood your idea. I'm sorry I had some misunderstanding about this...

I think that is OK with your opinion. I will make resolved. 🙇